packages/@aws-cdk/cdk/README.md

```diff
@@ -117,11 +117,19 @@ has a few features that are covered later to explain how this works.
 
 ### API
 
-In order to enable additional controls a Tags can specifically include or
+In order to enable additional controls a Tag can specifically include or
 exclude a CloudFormation Resource Type, propagate tags for an autoscaling group,
 and use priority to override the default precedence. See the `TagProps` 
 interface for more details. 
 
+Tags can be configured by using the properties for the AWS CloudFormation layer
```

Comment:
It should also be possible to specify tags in L2s via props (see #1935)

Reply:
Let's not complicate this PR even further, and update the readme when we implement #1935.

Reply:
Yes, just wanted to call that out